tests/testcases/core/db_classes/EE_CPT_Base_Test.php

```diff
@@ -118,6 +118,25 @@ public function testGetPrettyCurrentPostRemainingTheSame()
             "'{$event_with_reg_desc}' should not be the same as '{$other_event_desc}'"
         );
     }
+
+
+    /**
+     * @ticket 418 https://github.com/eventespresso/event-espresso-core/issues/418
+     */
+    public function testAddEventCategory()
+    {
+        $e = $this->new_model_obj_with_dependencies('Event');
+        /**
+         * @var $e EE_Event
+         */
+        $ee_term_taxonomy = $e->add_event_category(
+            'Rebel Ships',
+            'Events look at important Rebel Space Ships'
+        );
+        $this->assertTrue($ee_term_taxonomy instanceof EE_Term_Taxonomy);
+        $this->assertNotEquals(0, $ee_term_taxonomy);
+        $this->assertEquals('espresso_event_categories', $ee_term_taxonomy->taxonomy());
```

Comment:
as mentioned in my review in the other ticket, I don't think the second assertion is necessary.

Reply:
ah good points, I meant to do the first one but got dizzied by the rest of this UI. The second one I hadn't thought of but makes a ton of sense